desktop/libs/indexer/src/indexer/api3.py

```diff
@@ -117,7 +118,7 @@ def _convert_format(format_dict, inverse=False):
 def guess_format(request):
   file_format = json.loads(request.POST.get('fileFormat', '{}'))
 
-  if file_format['inputFormat'] == 'file':
+  if file_format['inputFormat'] == 'file' or file_format['inputFormat'] == 'localfile' :
```

Comment:
so can we do this because both are files?
but after doing guess_formate API giving error like "Expecting value: line 1 column 1 (char 0)"

Reply:
Obviously we would need to treat the upload separately first, so that we have the data. 

if file_format['inputFormat'] == 'localfile':
 ...